src/mgmt/cli_mgmt.c

```diff
@@ -56,63 +56,11 @@ dc_cp(tse_task_t *task, void *data)
 }
 
 int
-dc_mgmt_svc_rip(tse_task_t *task)
+dc_deprecated(tse_task_t *task)
 {
-	daos_svc_rip_t		*args;
-	struct cp_arg		 cp_arg;
-	crt_endpoint_t		 svr_ep;
-	crt_rpc_t		*rpc = NULL;
-	crt_opcode_t		 opc;
-	struct mgmt_svc_rip_in	*rip_in;
-	int			 rc;
-
-	args = dc_task_get_args(task);
-	rc = dc_mgmt_sys_attach(args->grp, &cp_arg.sys);
-	if (rc != 0) {
-		D_ERROR("failed to attach to grp %s, rc "DF_RC".\n",
-			args->grp, DP_RC(rc));
-		rc = -DER_INVAL;
-		goto out_task;
-	}
-
-	svr_ep.ep_grp = cp_arg.sys->sy_group;
-	svr_ep.ep_rank = args->rank;
-	svr_ep.ep_tag = daos_rpc_tag(DAOS_REQ_MGMT, 0);
-	opc = DAOS_RPC_OPCODE(MGMT_SVC_RIP, DAOS_MGMT_MODULE,
-			      DAOS_MGMT_VERSION);
-	rc = crt_req_create(daos_task2ctx(task), &svr_ep, opc, &rpc);
-	if (rc != 0) {
-		D_ERROR("crt_req_create(MGMT_SVC_RIP) failed, rc: "DF_RC".\n",
-			DP_RC(rc));
-		D_GOTO(err_grp, rc);
-	}
-
-	D_ASSERT(rpc != NULL);
-	rip_in = crt_req_get(rpc);
-	D_ASSERT(rip_in != NULL);
-
-	/** fill in request buffer */
-	rip_in->rip_flags = args->force;
-
-	crt_req_addref(rpc);
-	cp_arg.rpc = rpc;
-
-	rc = tse_task_register_comp_cb(task, dc_cp, &cp_arg, sizeof(cp_arg));
-	if (rc != 0)
-		D_GOTO(err_rpc, rc);
-
-	D_DEBUG(DB_MGMT, "killing rank %u\n", args->rank);
-
-	/** send the request */
-	return daos_rpc_send(rpc, task);
-
-err_rpc:
-	crt_req_decref(rpc);
-err_grp:
-	dc_mgmt_sys_detach(cp_arg.sys);
-out_task:
-	tse_task_complete(task, rc);
-	return rc;
+	D_ERROR("This API is deprecated "DF_RC"\n", DP_RC(-DER_NOSYS));
```

Comment:
Minor nit...I don't think we need to print the DF_RC here

Reply:
I did struggle on this, kind of over/dup stated, @cibervicho maybe in better position to remove it(instead of rerun CI) if all CI passed this time, that way we can land it sooner to give David more room to pick this PR up and make this small change.

Reply:
I just took care of this in my PR #4008:

`int
dc_deprecated(tse_task_t *task)
{
	D_ERROR("This API is deprecated\n");
	tse_task_complete(task, -DER_NOSYS);
	return -DER_NOSYS;
}`